src/braket/experimental/autoqasm/types/py_to_oqpy.py

```diff
@@ -73,107 +78,38 @@ def wrap_value(node: Any, **kwargs: dict) -> Any:
-def _(node: bool, **kwargs: dict):
+def _(node: bool):
     return oqpy.BoolVar(
         node, name=program.get_program_conversion_context().next_var_name(oqpy.BoolVar)
     )
 
 
 @wrap_value.register(int)
 @wrap_value.register(np.integer)
-def _(node: Union[int, np.integer], **kwargs: dict):
+def _(node: Union[int, np.integer]):
     return oqpy.IntVar(
         node, name=program.get_program_conversion_context().next_var_name(oqpy.IntVar)
     )
 
 
 @wrap_value.register(float)
 @wrap_value.register(np.floating)
-def _(node: Union[float, np.floating], **kwargs: dict):
+def _(node: Union[float, np.floating]):
     return oqpy.FloatVar(
         node, name=program.get_program_conversion_context().next_var_name(oqpy.FloatVar)
     )
 
 
 @wrap_value.register
-def _(node: Iterable, **kwargs: dict):
-    # TODO: pass base_type to ArrayVar constructor
-    return oqpy.ArrayVar(
-        node,
-        dimensions=list(np.shape(node)),
-        name=program.get_program_conversion_context().next_var_name(oqpy.ArrayVar),
-    )
-
-
-@wrap_value.register
-def _(node: oqpy.base.Var, **kwargs: dict):
+def _(node: oqpy.base.Var):
     return node
 
 
 @wrap_value.register
-def _(node: oqpy.base.OQPyExpression, **kwargs: dict):
+def _(node: oqpy.base.OQPyExpression):
     return node
 
 
 @wrap_value.register
-def _(node: python_ast.Constant, **kwargs: dict):
-    return node.value
-
-
-@wrap_value.register
-def _(node: python_ast.Name, **kwargs: dict):
-    return oqpy.classical_types.Identifier(node.id)
-
-
-@wrap_value.register
-def _(node: python_ast.BinOp, **kwargs: dict):
-    return wrap_value(node.op, left=node.left, right=node.right)
-
-
-@wrap_value.register
-def _(node: python_ast.Add, **kwargs: dict):
-    left = kwargs["left"]
-    right = kwargs["right"]
-    return oqpy.base.OQPyBinaryExpression(
-        getattr(qasm_ast.BinaryOperator, "+"),
-        wrap_value(left),
-        wrap_value(right),
-    )
-
-
-@wrap_value.register
-def _(node: python_ast.Sub, **kwargs: dict):
-    left = kwargs["left"]
-    right = kwargs["right"]
-    return oqpy.base.OQPyBinaryExpression(
-        getattr(qasm_ast.BinaryOperator, "-"),
-        wrap_value(left),
-        wrap_value(right),
-    )
-
-
-@wrap_value.register
-def _(node: python_ast.Call, **kwargs: dict):
-    if node.func.id == "range":
-        arg_values = [wrap_value(arg) for arg in node.args]
-        return range(*arg_values)
-
-
-@wrap_value.register
-def _(node: python_ast.Compare, **kwargs: dict):
-    if len(node.ops) > 1:
-        raise NotImplementedError(node)
-    op = node.ops[0]
-    op_map = {
-        python_ast.Eq: "==",
-        python_ast.NotEq: "!=",
-        python_ast.Lt: "<",
-        python_ast.LtE: "<=",
-        python_ast.Gt: ">",
-        python_ast.GtE: ">=",
-    }
-    return oqpy.base.OQPyBinaryExpression(
-        getattr(qasm_ast.BinaryOperator, op_map[type(op)]),
-        wrap_value(node.left),
-        wrap_value(node.comparators[0]),
-    )
```

Comment:
Why the removals?

Reply:
Unused code, and no known way to reach these code paths. I'd consider this unsupported.